src/lib/cli/format.ts

```diff
@@ -134,7 +134,8 @@ export function keyValue( values: Tuple[] ): string {
 export function requoteArgs( args: string[] ): string[] {
 	return args.map( arg => {
 		if ( arg.includes( '--' ) && arg.includes( '=' ) && arg.includes( ' ' ) ) {
-			return arg.replace( /^--([^=]*)=(.*)$/, '--$1="$2"' );
+			const quote = arg.includes( '"' ) ? "'" : '"';
```

Comment:
@chriszarate Perhaps we should replace all the non-terminal `"` by `/"`? Would that be a better idea?

Reply:
Maybe. I think I am unclear on what the "good" result is from this function. Maybe you could update the test with additional test cases that show what it is supposed to do? I mostly want to make sure that it doesn't provide a way to inject a command.